### PR TITLE
Create Automated branches in apiserver-network-proxy upstream repo

### PR DESCRIPTION
**Pull request by Basavaraju-G**
In order to include apiserver-network-proxy image into the OCP payload, we need to enable automatic
branch creation by adding OCP namespace 

References:
https://docs.ci.openshift.org/docs/architecture/branching/#how-do-i-opt-my-repository-into-automated-branching

### CODE REVIEW

**Review by mkumatag — APPROVED** [review]
/lgtm

File: core-services/image-mirroring/apiserver-network-proxy/OWNERS [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- enxebre
+- ironcladlou
+- sjenning
+options: {}
+reviewers:
+- csrwng
+- enxebre
+- ironcladlou
+- relyt0925
+- sjenning
```
Comment (bparees) [inline core-services/image-mirroring/apiserver-network-proxy/OWNERS:18]: these mirror files/directories are based on the repo org, so i'd expect this to be "image-mirroring/hypershift", given where it is pushing to, and that already exists.  Why not add it there?

File: core-services/image-mirroring/apiserver-network-proxy/mapping_apiserver-network-proxy_4_11 [hunk 2]
```diff
@@ -0,0 +1 @@
+registry.ci.openshift.org/ocp/4.11:apiserver-network-proxy quay.io/hypershift/apiserver-network-proxy:4.11 quay.io/hypershift/apiserver-network-proxy:latest
```
Comment (bparees) [inline core-services/image-mirroring/apiserver-network-proxy/mapping_apiserver-network-proxy_4_11:1]: given you're also putting it under the openshift org, why is it needed here?

Comment (bparees) [reply core-services/image-mirroring/apiserver-network-proxy/mapping_apiserver-network-proxy_4_11:1]: `kube-apiserver-network-proxy` would give better alignment to existing names, i think, unless this is generically useful for any apiserver.

Comment (alvaroaleman) [reply core-services/image-mirroring/apiserver-network-proxy/mapping_apiserver-network-proxy_4_11:1]: I think this is autogenerated. Renaming it sgtm, although it isn't really specific to the kube-apiserver but used by anything that needs to access the pod or service network. The current name is simply the name of the upstream project: https://github.com/kubernetes-sigs/apiserver-network-proxy

Comment (bparees) [reply core-services/image-mirroring/apiserver-network-proxy/mapping_apiserver-network-proxy_4_11:1]: autogenerated from what?

i'm still not sure why we'd mirror the same image twice, to two different repo orgs.

Comment (mkumatag) [reply core-services/image-mirroring/apiserver-network-proxy/mapping_apiserver-network-proxy_4_11]: IMO: mirroring into hypershift namespace may not needed after we ship into the release payload I feel.

@Basavaraju-G can you please remove the entry from the hypershift mirror and rerun the make to see if it still adds an entry, if yes - we need to find what is that config which is adding this automated entry.

Comment (Basavaraju-G) [reply core-services/image-mirroring/apiserver-network-proxy/mapping_apiserver-network-proxy_4_11]: that worked manju. it added to only ocp mapping not hypershift one.

**Comment by mkumatag** [comment]
@Basavaraju-G can you please squash the commits?

**Comment by mkumatag** [comment]
All the comments addressed and CI is also green, @bparees ptal

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**